*** testsBoth-EF-Submissao/testsBoth/A-026-026-M-ok.out	2023-10-19 17:31:45.089374000 +0100
--- testsBoth-EF-Submissao/testsBoth/A-026-026-M-ok.outhyp	2023-10-28 00:27:31.376370548 +0100
***************
*** 29,36 ****
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
! Valor a procurar: 49
! 1;2|49
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
--- 29,40 ----
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 1
! Menu de Consultas
! 1 - Procurar valores
! 2 - Procurar funções
! 0 - Sair
! Escolha uma opção: 49
! Opção inválida!
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
