*** testsBoth-EF-Submissao/testsBoth/A-031-031-M-ok.out	2023-10-19 17:31:45.314567000 +0100
--- testsBoth-EF-Submissao/testsBoth/A-031-031-M-ok.outhyp	2023-10-28 00:27:31.588369364 +0100
***************
*** 29,40 ****
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 2
- Função a procurar: ADD
- 2;3|48=ADD(2;2,5)
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 0
  Menu Principal
  1 - Novo
--- 29,40 ----
  2 - Procurar funções
  0 - Sair
  Escolha uma opção: 2
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
+ Escolha uma opção: ADD
+ Número inválido!
  Escolha uma opção: 0
  Menu Principal
  1 - Novo
